patches/nix-integration.patch

```diff
@@ -74,50 +74,33 @@ index 4335f8db1..78d2817c1 100644
  
  ;;
 diff --git a/lisp/doom.el b/lisp/doom.el
-index 6dbbefb5e..6116445f3 100644
+index 6dbbefb5e..2fc52ffb7 100644
 --- a/lisp/doom.el
 +++ b/lisp/doom.el
-@@ -221,7 +221,7 @@ downloaded/installed by packages. Must end in a slash.")
-       (expand-file-name (file-name-as-directory localdir))
-     (if doom-profile
-         doom-profile-dir
--      (expand-file-name ".local/" doom-emacs-dir)))
-+      "@local@/"))
-   "Root directory for local storage.
- 
- Use this as a storage location for this system's installation of Doom Emacs.
-@@ -229,10 +229,11 @@ Use this as a storage location for this system's installation of Doom Emacs.
+@@ -229,10 +229,13 @@ Use this as a storage location for this system's installation of Doom Emacs.
  These files should not be shared across systems. By default, it is used by
  `doom-data-dir' and `doom-cache-dir'. Must end with a slash.")
  
-+; nix-doom-emacs: This doesn't meet XDG but backwards compatibility is a thing.
++(defconst nix-doom-local-dir (expand-file-name "~/.local/share/doom/")
++  "Root directory for transient files. Equivalent to doom-local-dir.")
```

Comment:
If we are going to use this only for `doom-data-dir` them I think it is better to not introduce this (and make RSS module go back to concat with `doom-cache-dir`).

Reply:
Done.